tldr/cross_file_calls.py

```
@@ -3939,3 +3943,266 @@ def _build_php_call_graph(
+                    defined_names.add(f"{current_type}.{name}")
+        for child in node.children:
+            collect_definitions(child)
+
+    collect_definitions(tree.root_node)
+
+    def _extract_call_from_expr(call_node):
+        for child in call_node.children:
+            if child.type == "simple_identifier":
+                callee_name = source[child.start_byte:child.end_byte].decode("utf-8")
+                return ("bare", callee_name)
+            elif child.type == "navigation_expression":
```

Ruff (0.15.12)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)